File: csharp/src/Ice/SSL/EndpointInfo.cs
// Copyright (c) ZeroC, Inc.

#nullable enable

namespace Ice.SSL;

/// <summary>
/// Provides access to an SSL endpoint's information.
/// </summary>
public sealed class EndpointInfo : Ice.EndpointInfo
{
    internal EndpointInfo(Ice.EndpointInfo underlying)
        : base(underlying)
    {
    }
}

File: matlab/lib/+Ice/+SSL/EndpointInfo.m
classdef (Sealed) EndpointInfo < Ice.EndpointInfo
    %ENDPOINTINFO Provides access to SSL endpoint information.

    % Copyright (c) ZeroC, Inc.

    methods(Hidden)
        function obj = EndpointInfo(underlying)
            assert(nargin == 1, 'Invalid number of arguments');
            obj@Ice.EndpointInfo(underlying);
        end
    end
end
